zer0-os/zOS
# feat: confirm reset password flow

### What does this do?
- adds confirm reset password flow
- wires up backend end point to reset the password with auth0

### Why are we making this change?
- to allow a user to reset their password on the `set-password` route.

### How do I test this?
- run the app and run local api > go to log in screens > select `reset` in the footer when logging in via email > enter email address > open email and click link > enter new password > confirm new password > log in with new password.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?

<img width="1937" alt="Screenshot 2023-08-18 at 16 40 59" src="https://github.com/zer0-os/zOS/assets/39112648/2eba90d1-17a4-4232-bb84-0672d83325a5">

<img width="1937" alt="Screenshot 2023-08-18 at 16 45 06" src="https://github.com/zer0-os/zOS/assets/39112648/37f8a6c1-d0d3-47b7-8343-deec1005b002">

<img width="1937" alt="Screenshot 2023-08-18 at 16 45 11" src="https://github.com/zer0-os/zOS/assets/39112648/dc13c45e-177a-4b25-8a2a-f81ac73d7abd">

<img width="1937" alt="Screenshot 2023-08-18 at 16 45 29" src="https://github.com/zer0-os/zOS/assets/39112648/bda21546-53c9-41c3-b977-e34b8cff7c63">

<img width="1937" alt="Screenshot 2023-08-18 at 16 41 14" src="https://github.com/zer0-os/zOS/assets/39112648/d13b8092-ee8b-4dcc-979a-0f6c33b9f44a">


<img width="1937" alt="Screenshot 2023-08-18 at 16 42 53" src="https://github.com/zer0-os/zOS/assets/39112648/cc35f2d0-6a65-4f34-989b-738470a9e532">
